### ISSUE TITLE
Exporting plots to PNG from within Java

### ISSUE THREAD

**Issue by kars0414** [jtablesaw/tablesaw#678]
Guys, am making beautiful interactive plots with Plot.show() - love tablesaw so far - but I also need to autogenerate a static PNG image version that can be emailed.

I can't see how this is achieved from within Java ?  Am I missing something very basic ?

**Comment by lwhite1** [comment]
You can only create a png from the rendered JavaScript. Plotly added
support for direct rendering recently, but we haven’t added it yet. I was
looking for the same functionality myself, but it seems non-trivial to do
from java

**Comment by doronl** [comment]
any news with this requrest?

**Comment by lwhite1** [comment]
i don't expect to work on this in the near future.

**Comment by mrusignola** [comment]
Any update on this request?

**Comment by lwhite1** [comment]
No. It’s very difficult to do this. It needs a JavaScript engine to produce
the plot.
